Makefile

```diff
@@ -1,5 +1,16 @@
+PREFIX?=/usr/local
+CP=cp
+MKDIR=mkdir
+
 build:
 	@sh scripts/build.sh
 
 test:
 	@sh scripts/test.sh
+
+swiftpm:
+	CONFIGURATION=release ./scripts/build.sh
+
+swiftpm_install: swiftpm
+	-$(MKDIR) "$(PREFIX)/bin"
+	$(CP) -f ./.build/release/grimoire "$(PREFIX)/bin/"
```

Comment:
Im unsure about these names (`swiftpm` and `swiftpm_install`). Are you using them because they are respecting some form of convention? I would rename them like verbs:
```
swiftpm -> build_release
swiftpm_install -> install???
```
What do you think?

Reply:
I kind of stole it from Carthage, but I agree with you.